lib/generators/hyrax/templates/config/initializers/hyrax.rb

```diff
@@ -143,6 +143,9 @@
   # Returns a IIIF image size default
   # config.iiif_image_size_default = '600,'
 
+  # Fields to display in the IIIF metadata section; default is the required fields
+  # config.iiif_metadata_default = Hyrax::Forms::WorkForm.required_fields
```

Comment:
Would you mind renaming this to `iiif_metadata_fields`?

Reply:
Done